packages/blockchain-api/src/events/EscrowReceived.ts

```diff
@@ -7,8 +7,10 @@ import { Contracts } from '../utils'
 export class EscrowReceived extends TransactionType {
   matches(transaction: Transaction): boolean {
     return (
-      transaction.transfers.length === 1 &&
-      transaction.transfers.containsTransferFrom(Contracts.Escrow)
+      (transaction.transfers.length === 1 &&
+        transaction.transfers.containsTransferFrom(Contracts.Escrow)) ||
+      (transaction.transfers.length === 2 &&
+        transaction.transfers.containsAddressToEqualFromAdress(Contracts.Escrow))
```

Comment:
I think we could keep the logic of `TransferNavigator` generic enough without making methods too specific. You could just check that there are two transfers and that there is a transfer from the escrow contract and there is a transfer to the wallet address (account hash). In this case we can drop the `containsAddressToEqualFromAdress` altogether.